src/Componente/ListaCuloriAdaugate.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+const ListaCuloriAdaugate = (props) => {
+  const { listaCuloare, setListaCuloare } = props;
+
+  const handleDeleteFromList = (colorName) => {
+    console.log("NUMBE PE CARE DAU CLICK", colorName);
```

Comment:
Partea de debugging e de preferat :)) sa nu le lasi in commit...Poti sa te trezesti ca lasi niste cuvinte =)) pe care nu ai vrea ca colegii tai sa le vada.